src/components/pages/Categories/List.vue

```diff
@@ -33,10 +38,33 @@ export default {
     categoryList() {
       return this.$store.state.categories.categoryList;
     },
+    // POST
+    categoryName() {
+      const { name } = this.$store.state.categories.targetCategory;
+      return name;
+    },
+    doneMessage() {
+      return this.$store.state.categories.doneMessage;
+    }, // 作成成功時のメッセージ
   },
   created() {
     this.$store.dispatch('categories/getAllCategories');
   },
+  methods: {
+    handleSubmit() {
+      if (this.loading) return;
+      this.$store.dispatch('categories/postCategory').then(() => {
+        this.$router.push({
+          path: '/categories',
+          query: { redirect: '/categories' },
+        });
+      });
+    },
+    updateValue($event) {
+      this.$store.dispatch('categories/updateValue', $event.target.value); // actionsのupdateValueに入力値を渡す
```

Comment:
[imo]

プロジェクトの方針にもよりますが、「処理の説明」をしているコメントアウトは基本的に記載不要かなと個人的には思います。
コードを見れば、何をやっているか（What）はわかる人には伝わるためです。
もしコメントアウトを残しておきたいのであれば、なぜその処理が必要だったか（Why）を書いておくと、意図も伝わるので良いかなと思います。
他の箇所も見てみてください。

```suggestion
      this.$store.dispatch('categories/updateValue', $event.target.value);
```

Reply:
a82aa8ed10fffbfd2d1e4999cd4837fbc7117aa0
コメントアウトを削除しました。以後残す場合はWhyを意識し書いてみます